cmake/configure/configure_adolc.cmake

```diff
@@ -68,6 +68,8 @@ ENDMACRO()
 MACRO(FEATURE_ADOLC_CONFIGURE_EXTERNAL)
   SET(DEAL_II_ADOLC_WITH_ATRIG_ERF ${ADOLC_WITH_ATRIG_ERF})
   SET(DEAL_II_ADOLC_WITH_ADVANCED_BRANCHING ${ADOLC_WITH_ADVANCED_BRANCHING})
+
+  SET(DEAL_II_EXPAND_ADOLC_TYPES "::adouble; adtl::adouble")
```

Comment:
i though this part was merged already. Maybe rebase?

Reply:
Yes, I do need to do a rebase on this branch. I'll see if I can get around to doing so in the next couple of days.